dungeon/src/starter/Starter.java

```diff
@@ -213,14 +208,19 @@ private static void taskSelectorOnSetup(Set<DSLEntryPoint> entryPoints) {
         });
   }
 
-  private static Set<DSLEntryPoint> processCLIArguments(String[] args)
-      throws IOException, ParseException {
-    Set<DSLEntryPoint> entryPoints = new HashSet<>();
-    DSLEntryPointFinder finder = new DSLEntryPointFinder();
-    DSLFileLoader.processArguments(args)
-        .forEach(path -> finder.getEntryPoints(path).ifPresent(entryPoints::addAll));
-    if (entryPoints.isEmpty()) throw new ParseException("no entry points found", 0);
-    return entryPoints;
+  private static Set<DSLEntryPoint> processCLIArguments(String[] args) throws IOException {
+    try {
+      Set<DSLEntryPoint> entryPoints = new HashSet<>();
+      DSLEntryPointFinder finder = new DSLEntryPointFinder();
+      DSLFileLoader.processArguments(args)
+          .forEach(path -> finder.getEntryPoints(path).ifPresent(entryPoints::addAll));
+      if (entryPoints.isEmpty()) {
+        throw new IOException("No entry points found.");
+      }
+      return entryPoints;
+    } catch (IOException e) {
+      throw new IOException(String.join(" ", "Couldn't open specified .dng.", e.getMessage()));
+    }
```

Comment:
Hmmm, das hier ist doppelt-gemoppelt. Die neue IOE in Zeile 218 wird direkt in Zeile 221 gefangen und durch eine (dann falsche) Fehlermeldung ergänzt.

Ich denke, in Zeile 218 war die frühere ParseException schon ganz gut.

Reply:
💭 ok, mache ich rückgängig.

```suggestion
        throw new ParseException("entry points empty", 0);
      }
      return entryPoints;
    } catch (IOException e) {
      throw new IOException(String.join(" ", "Couldn't open specified .dng.", e.getMessage()), e);
    }
```

Reply:
nimm bitte auch den alten string mit ... `"No entry points found."`

Reply:
Aye aye 😉 🖖🏼